sdk/cognitivelanguage/ai-language-conversations/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```

Comment:
could you please use a standard tsconfig.json? You can see an example in the ai-language-text library.